paddle/fluid/eager/grad_node_info.h

```diff
@@ -95,8 +95,12 @@ class GradNodeBase {
    * is better choice to fit this format.
    * **/
   virtual std::vector<std::vector<paddle::experimental::Tensor>> operator()(
-      const std::vector<std::vector<paddle::experimental::Tensor>>& grads) = 0;
+      const std::vector<std::vector<paddle::experimental::Tensor>>& grads,
+      const bool create_graph = false) = 0;
 
+  virtual void ClearTensorWrappers() = 0;
+
+  virtual bool IsClearTensorWrappers() = 0;
```

Comment:
Just nitpicking, better named as "IsTensorWrappersCleared()" or "IsEmptyTensorWrappers()". 

"clear" as an adj. doesn't actually mean what you expected: 

<img width="675" alt="image" src="https://user-images.githubusercontent.com/22334008/158096020-fdb7fd6f-f7f0-45a3-9f3b-225b15a1391e.png">

Reply:
Thx, use IsTensorWrappersCleared() now.